### ISSUE TITLE
Coverage.py incompatible with pyratemp

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#221]
*Originally reported by* **Anonymous**

----------------------------------------

Coverage.py doesn't work with the pyratemp library that can be found here: http://www.simple-is-better.org/template/pyratemp.html

I did setup a small test case:

```
template.txt:
Hello @!thing!@!

test.py:
import pyratemp

t = pyratemp.Template(filename="template.txt")
print t(thing="World")

pyratemp.py:
http://www.simple-is-better.org/template/pyratemp-current/pyratemp.py
```

Running python-coverage run test.py works, but when running python-coverage html in the same directory, this error is shown:

```
No source for code: '/home/marten/Bureaublad/coverage-test': [Errno 21] Is a directory: '/home/marten/Bureaublad/coverage-test'
```

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/221

**Comment by nedbat** [comment]
Fixed in <<changeset [803b614a6ac2 (bb)](https://bitbucket.org/ned/coveragepy/commits/803b614a6ac2)>>.

**Comment by nedbat** [comment]
*Original comment by* **Marten de Vries (Bitbucket: [marten-de-vries](https://bitbucket.org/marten-de-vries), GitHub: [marten-de-vries](https://github.com/marten-de-vries))**

----------------------------------------

That's a very fast fix. Thanks!
